receiver/prometheusreceiver/metrics_receiver_open_metrics_test.go

```diff
@@ -84,7 +84,12 @@ func verifyNegativeTarget(t *testing.T, td *testData, mds []pmetric.ResourceMetr
 
 	require.Greater(t, len(mds), 0, "At least one resource metric should be present")
 	metrics := getMetrics(mds[0])
-	assertUp(t, 0, metrics)
+
+	// There are two possibilities here
+	// 1. The Prometheus scrape parser rejected the sample as invalid. In this case, we get no metrics.
+	// 2. The Prometheus scrape parser accepted the sample, but the receiver dropped it due to incompatibility with the Otel schema.
```

Comment:
Should we just make case 2 a "positive" case?  Or should we separate into three verify function?

Reply:
Logically, it seems "negative" to me, in the sense that we don't actually ingest the metric. If anything the "positive" test should check that any non-default metric is present.

Reply:
Could we split the negative cases into fail vs invalid then?

Reply:
Allright, I renamed the "invalid" test cases appropriately and added a separate test for them. The rest stays the same, although I've also added the additional check to positive tests - they all pass regardless.